pySDC/tests/test_projects/test_resilience/test_extrapolated_error_within_Q.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from pySDC.projects.Resilience.extrapolation_within_Q import check_order
+from pySDC.projects.Resilience.advection import run_advection
+from pySDC.projects.Resilience.piline import run_piline
```

Comment:
Please add the imports to the test function, because otherwise MPI doesn't like it.